### ISSUE TITLE
Which env variables are required ?

### ISSUE THREAD

**Issue by NicoPy** [AdaCore/ada_language_server#1137]
When launched from **outside** a Alire terminal ALS shows `GPR PATH: ` at startup.
When launched from a Alire terminal, ALS shows `GPR_PATH:xxx` whith _xxx_ set to the content of `GPR_PROJECT_PATH` env variable.
Is it required to set `GPR_PROJECT_PATH` env variable ?

Also, are other env variables set by Alire required ?

**Comment by reznikmm** [comment]
ALS checks `ALIRE=True` env to see if alire was activated. If so it skips `alr` execution. Beside this only `gprbuild` related variables (like `GPR_PROJECT_PATH`) are used.

**Comment by NicoPy** [comment]
In the case that `ALIRE` is not set but an Alire project is detected (ALS run outside Alire terminal), does ALS automatically set `GPR_PROJECT_PATH` ?

**Comment by reznikmm** [comment]
@NicoPy Yes, ALS sets the `GPR_PROJECT_PATH`, but this settings is visible only in subprocess of ALS, such as `gnatls`. It doesn't influence VS Code terminals or debugger for example, because they are launched by VS Code, not by ALS itself. Auto-detected tasks (such as **Build current project**, or **Check current file**) were modified to take `alr` environments into account.

**Comment by NicoPy** [comment]
I'm trying to find a solution to a problem I have : I'm trying to use ALS with Eclipse editor. I'm able to make it work correctly with __one__ Alire project (by running Eclipse from an Alire terminal). But as Eclipse is a multi-project editor, I'd like that each project uses it's own ALS configuration. That is currently not possible. ALS being able to determine it's configuration from an Alire project (when env variables are not set) would be a solution. 

Note : To be more specific, the problem is related to the ALS search path for auto-completion/documentation. With multiple projects, the ALS search path is not project specific. 
I know ALS has been designed for a single project editor but I have use cases where a multi-project editor is really helpful. For example when designing a system where part of the software runs on a PC and another part runs of an embedded target.

**Comment by AnthonyLeonardoGracio** [comment]
Hello @NicoPy, 

Why not spawn multiple ALS instances, one per project?
